sos/report/plugins/microshift.py

```diff
@@ -26,7 +27,7 @@ class Microshift(Plugin, RedHatPlugin):
     short_desc = 'Microshift'
     plugin_name = 'microshift'
     plugin_timeout = 900
-    packages = ('microshift', 'microshift-selinux', 'microshift-networking')
+    packages = ('microshift', 'microshift-selinux', 'microshift-networking', 'microshift-greenboot')
```

Comment:
Please split this into multiple lines to adhere to PEP8.

Reply:
Done.